src/widgets/Notebook.hpp

```diff
@@ -35,6 +38,7 @@ class Notebook : public BaseWidget
     int indexOf(QWidget *page) const;
     virtual void select(QWidget *page, bool focusPage = true);
     void selectIndex(int index, bool focusPage = true);
```

Comment:
It would be nice if `selectIndex` was made private, but we use it in the SelectChannelPopup